src/main/java/org/osc/sdk/controller/api/SdnRedirectionApi.java

```diff
@@ -72,9 +76,13 @@
     List<NetworkElement> getNetworkElements(NetworkElement element) throws Exception;
 
     /**
+     * <p>
      * Creates an inspection hook on SDN controller, see {@link InspectionHookElement}
+     * In case of NSC and Nuage, inspectionPort represents redirection target's egress, ingress port ids
```

Comment:
For example, inspectionPort represents redirection target's egress, ingress port ids
In case of SFC, inspectionPort **parent id** is expected to provide a port chain id
This is based on the design document.
> This interfaces parentId argument carries either NULL in case of no port chain is created at target Openstack nodes or an already existing port chain Id to add more flow classifiers to that port chain ID.

What is the identifier string expected to be returned as part of installinspectionhook?